README.md

```diff
@@ -0,0 +1,66 @@
+The canonical URL for the upstream Spectrum git repository is
+<https://spectrum-os.org/git/spectrum/>.
+
+To try Spectrum OS, you need to build it from source and then install it to an SD card.
+
+> Cross-compiled version of Spectrum OS is for i.MX 8QXP board only.
+
+## Building a Spectrum OS Image
```

Comment:
Please refactor "## Setting up binary caches (optional, recommended)"-chapter and put it before ## Building a Spectrum OS Image

Reply:
Done.